build/webpack.config.umd.ts

```diff
@@ -1,36 +1,39 @@
 import config from '../config'
 import webpackConfig from '../webpack.config'
 
-const pkg = require('../package.json')
 const { paths } = config
 
-const webpackUMDConfig = {
-  target: 'web',
-  mode: 'production',
-  devtool: false,
-  entry: {
-    [pkg.name]: paths.src('umd.ts'),
-  },
-  externals: {
-    react: 'React',
-    'react-dom': 'ReactDOM',
-  },
-  output: {
-    filename: 'stardust-ui-react.min.js',
-    libraryTarget: 'umd',
-    library: 'Stardust',
-    path: paths.dist('umd'),
-    publicPath: '/',
-    pathinfo: true,
-  },
-  resolve: webpackConfig.resolve,
-  module: {
-    noParse: webpackConfig.module.noParse,
-    rules: webpackConfig.module.rules,
-  },
-  performance: {
-    hints: false, // to (temporarily) disable "WARNING in entrypoint size limit: The following entrypoint(s) combined asset size exceeds the recommended limit")
-  },
+const webpackUMDConfig = (packageName: string) => {
+  const pkg = require(`../packages/${packageName}/package.json`)
+
+  return {
+    target: 'web',
+    mode: 'production',
+    devtool: false,
+    entry: {
+      [pkg.name]: paths.packageSrc(packageName, 'umd.ts'),
+    },
+    externals: {
+      react: 'React',
+      'react-dom': 'ReactDOM',
+    },
+    output: {
+      filename: 'stardust-ui-react.min.js',
```

Comment:
we should reflect package name here as well
```js
`stardust-ui-${packageName}.min.js`
```

Reply:
👍